src/transport_ofi.c

```diff
@@ -1197,6 +1223,17 @@ void shmem_transport_print_info(void)
            (NULL != shmem_util_getenv_str("OFI_ATOMIC_CHECKS_WARN")) ?
            "Set" : "Not set");
     printf("\tDisplay warnings about unsupported atomic operations\n");
+
+    if (NULL != shmem_util_getenv_str("OFI_GLOBAL_POLL_LIMIT")) {
+        printf("SMA_OFI_GLOBAL_POLL_LIMIT      %ld\n", shmem_util_getenv_long("OFI_GLOBAL_POLL_LIMIT", 0, DEFAULT_POLL_LIMIT));
+        printf("\tMax # of polls for put/get completions is %"PRIu64"\n", shmem_transport_ofi_put_poll_limit);
```

Comment:
I think we want to handle this one differently, because if it's set it will override the two below.  However, if it's not set, we will still print the default poll limit.